src/langcheck/metrics/eval_clients/__init__.py

```diff
@@ -1,8 +1,10 @@
+from langcheck.metrics.eval_clients._anthropic import AnthropicEvalClient
```

Comment:
(Related to comment on deps) If we don't put `anthropic` in the core deps, we wrap this in a try/except